### PR TITLE
Add ideas 300 to 306, corresponding to teams active

### PR DESCRIPTION
**Pull request by oskarth**
Updates teams with rough skeleton descriptions and contributors so we can have a clearer view of https://ideas.status.im/people and https://ideas.status.im/all

Addresses https://github.com/status-im/ideas/issues/305

Please look over your teams and see what changes you want to make. Can also do changes in separate PR if this seems more appropriate

<blockquote><div>Status - Ideas</div><div><strong><a href="http://ideas.status.im/people">People</a></strong></div><div>Swarm Home. New, completed and in-progress features for Status.</div></blockquote>
<blockquote><div>Status - Ideas</div><div><strong><a href="http://ideas.status.im/all">Ideas</a></strong></div><div>Swarm Home. New, completed and in-progress features for Status.</div></blockquote>

The idea is also to get a better understanding of headcount and what each team is focusing on.

### CODE REVIEW

File: ideas/303-core-desktop.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+title: Core Desktop Team
+status: Active
+created: 2018-05-01
+category: core
+lead-contributor: vitaliy
+contributors:
+    - chadyj
+    - vitaliy
+    - maxris
+    - volodymyr
```
Comment (siphiuel) [inline ideas/303-core-desktop.md]: Please also add gnl as a core contributor.

File: ideas/302-core-mobile-app.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+id: 302-core-mobile-app-team
+title: Core Mobile App Team
+status: Active
+created: 2018-05-01
+category: core
+lead-contributor: mandrigal
```
Comment (mandrigin) [inline ideas/302-core-mobile-app.md]: typo: should be mandrigin

File: ideas/302-core-mobile-app.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+contributors:
+    - chadyj
+    - mandrigal
```
Comment (mandrigin) [inline ideas/302-core-mobile-app.md]: typo: should be mandrigin

File: ideas/306-core-dapp-developers.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+id: 306-core-dapp-team
+title: Core Dapp Team
+status: Active
+created: 2018-05-01
+category: core
+lead-contributor: jeluard
+contributors:
+    - jeluard
+    - flexsurfer
```
Comment (jeluard) [inline ideas/306-core-dapp-developers.md]: Can we add alwx too?

**Pull request merged**